### ISSUE TITLE
Date Between

### ISSUE THREAD

**Issue by chrishough** [ffaker/ffaker#256]
Have we considering adding the between for dates like Faker?
```

# Random date between dates
Faker::Date.between(2.days.ago, Date.today) #=> "Wed, 24 Sep 2014"

# Randome date between dates except for certain date
Faker::Date.between_except(1.year.ago, 1.year.from_now, Date.today) #=> "Wed, 24 Sep 2014"

# Random date in the future (up to maximum of N days)
Faker::Date.forward(23) # => "Fri, 03 Oct 2014"

# Random date in the past (up to maximum of N days)
Faker::Date.backward(14) #=> "Fri, 19 Sep 2014"
```

**Comment by PapePathe** [comment]
+1 for this.

**Comment by brnrdog** [comment]
I opened a PR to cover this feature (#263), I'm just waiting for a response.

**Comment by rocknruby** [comment]
@chrishough #between is now implemented and just merged into the master with #263 - I close this issue right now.
